launch-tests.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * Initialize the test suite by first launching the mock api server, and when
+ * it's up, fire off `testem ci`, which will run the test suite in up to five
+ * browsers concurrently. cha-ching!
+ */
+
+'use strict';
+var server;
+var testem;
+
+var cp = require('child_process');
+var errorDie = function(err) {
+    var code = err.code === undefined ? 1 : err.code;
+    console.log('Failed to start child process', err);
+    process.exit(code);
+};
+var closeChild = function(code) {
+    if (code > 0) {
+        errorDie({ message: 'mock-api-server exited', code: code });
+    }
+};
+var testemExit = function(code) {
+    console.log('test run complete, closing server (exit code ' + code + ')');
+    server.kill();
+    return closeChild(code);
+};
+var spawnTestem = function() {
+    testem = cp.spawn('testem', ['ci', '-p', '7575', '-P', '5'], { stdio: 'inherit' });
```

Comment:
`-P 5` => 5 concurrent browsers, `stdio: 'inherit'` => make the child process' stdxxx pipe directly into the parent's